lib/LANraragi/Controller/Api/Tankoubon.pm

```diff
@@ -92,6 +93,24 @@ sub update_archive_list {
     }
 }
 
+sub update_metadata {
```

Comment:
I'm unconvinced we need a new API endpoint here -- Couldn't the metadata json be another, optional parameter in the original update tankoubon endpoint?  

(I'm not a fan of the routing for the original endpoint through.. see comment below)

Reply:
We can, I just wanted to separate the metadata update from the content, but I can join them without problem.